go/vt/vtctl/reparentutil/util.go

```diff
@@ -268,6 +244,18 @@ func findCandidateAnyCell(
 	return nil
 }
 
+// getTabletsWithPromotionRules gets the tablets with the given promotion rule from the list of tablets
+func getTabletsWithPromotionRules(tablets []*topodatapb.Tablet, rule promotionrule.CandidatePromotionRule) []*topodatapb.Tablet {
+	var res []*topodatapb.Tablet
```

Comment:
fun thing about named return values, is you can do this:

```go
func getTablets(param T1, ...) (res []*topodatapb.Tablet) {
    for _, candidate := range tablets {
        if promoRule == rule {
            res = append(res, candidate)
        }
    }
    return res
}
```

and save on that declaration line

Reply:
Fixed